File: EcoSwapAndroid/src/main/java/com/darrenthiores/ecoswap/android/presentation/utils/ItemCategoryExtension.kt
package com.darrenthiores.ecoswap.android.presentation.utils

import com.darrenthiores.ecoswap.android.R
import com.darrenthiores.ecoswap.domain.item.model.ItemCategory

fun ItemCategory.getIcon(): Int {
    return when (id) {
        "1" -> R.drawable.clothing_icon
        "2" -> R.drawable.electronics_icon
        "3" -> R.drawable.books_icon
        "4" -> R.drawable.furniture_icon
        "5" -> R.drawable.gaming_icon
        "6" -> R.drawable.sport_icon
        "7" -> R.drawable.edible_icon
        "8" -> R.drawable.living_icon
        else -> R.drawable.ic_no_picture
    }
}

File: EcoSwapiOS/EcoSwapiOS/Presentation/Utils/ItemCategoryExtension.swift
//
//  ItemCategoryExtension.swift
//  EcoSwapiOS
//
//  Created by Darren Thiores on 02/01/24.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation
import shared

extension ItemCategory {
    func getImageName() -> String {
        switch id {
        case "1":
            return "clothing_icon"
        case "2":
            return "electronics_icon"
        case "3":
            return "books_icon"
        case "4":
            return "furniture_icon"
        case "5":
            return "gaming_icon"
        case "6":
            return "sport_icon"
        case "7":
            return "edible_icon"
        case "8":
            return "living_icon"
        default:
            return "ic_no_picture"
        }
    }
}
